### PR TITLE
fix: updated Select key to take in id

### PR DESCRIPTION
**Pull request by jenesh**
### PR Checklist

- [ ] Related to Abstract designs:
- [ ] Related to JIRA ticket: ABC-123
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### Description

We're making these changes so the select component can take in an optional `id` to make the keys unique.

Screen Shot 2020-09-17 at 1 32 35 PM

### Notes

Testing for `Select` on the way!

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.91e28e.0<br/>@codecademy/gamut-labs@4.0.4-alpha.91e28e.0<br/>@codecademy/gamut@16.0.5-alpha.91e28e.0<br/></pre>

File: packages/gamut-labs/src/lib/breakpointHooks/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { mapValues } from 'lodash';
-import { breakpoints as originalBreakpoints } from '@codecademy/gamut-styles/utils/variables';
+import { breakpoints as originalBreakpoints } from '@codecademy/gamut-styles';
```
Comment (codecaaron) [inline packages/gamut-labs/src/lib/breakpointHooks/index.ts:2]: This import is broken now.

**Review by codecaaron — APPROVED** [review]
LGTM

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.0b2a93.0<br/>@codecademy/gamut-labs@4.1.2-alpha.0b2a93.0<br/>@codecademy/gamut@16.0.5-alpha.0b2a93.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f695687c0de824f4df48407--gamut-preview.netlify.app>https://5f695687c0de824f4df48407--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f695687c0de824f4df48407>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.bf0042.0<br/>@codecademy/gamut-labs@4.1.2-alpha.bf0042.0<br/>@codecademy/gamut@16.0.5-alpha.bf0042.0<br/></pre>

File: packages/gamut/src/Form/__tests__/Select-test.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Select } from '../Select';
+
+describe('<Select>', () => {
```
Comment (codecaaron) [inline packages/gamut/src/Form/__tests__/Select-test.tsx:5]: Generally for components we just use their names / function reference and omit the `<` `>`
IE:
`describe('Select', () => {` or `describe(Select, () => {`

File: packages/gamut/src/Form/Select.tsx [hunk 3]
```diff
@@ -18,16 +18,20 @@ export const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
...
+        );
+      });
     } else if (isObject(options)) {
-      each(options, (val, key) => {
+      each(options, (text, val) => {
+        const key = id ? `${id}-${val}` : val;
         selectOptions.push(
-          <option key={key} value={key}>
-            {val}
+          <option key={key} value={val} data-testid={key}>
```
Comment (codecaaron) [inline packages/gamut/src/Form/Select.tsx:33]: Nice test ID addition!

**Review by codecaaron — APPROVED** [review]
Only a nit, but this looks great to me!

Thank you for the test coverage!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.d450e0.0<br/>@codecademy/gamut-labs@4.1.2-alpha.d450e0.0<br/>@codecademy/gamut@16.0.5-alpha.d450e0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f6a207294018d00a29d14c8--gamut-preview.netlify.app>https://5f6a207294018d00a29d14c8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f6a207294018d00a29d14c8>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.3810fb.0<br/>@codecademy/gamut-labs@4.1.2-alpha.3810fb.0<br/>@codecademy/gamut@16.0.5-alpha.3810fb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f6a228684b3490093d43342--gamut-preview.netlify.app>https://5f6a228684b3490093d43342--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f6a228684b3490093d43342>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.4.2-alpha.8e70a1.0<br/>@codecademy/gamut-labs@4.1.3-alpha.8e70a1.0<br/>@codecademy/gamut@16.0.6-alpha.8e70a1.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f6a25cec8fb3a00cfc67b72--gamut-preview.netlify.app>https://5f6a25cec8fb3a00cfc67b72--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f6a25cec8fb3a00cfc67b72>Deploy Logs</a></p>

**Pull request merged**